### ISSUE TITLE
Suggestion: Another representation of Super Quick Info

### ISSUE THREAD

**Issue by laicasaane** [wmjordan/Codist#216]
As we can see in these images, the methods are shown in a very clustered style thus hard to comprehend at first glance.

![image](https://user-images.githubusercontent.com/1594982/190428094-8cf084ba-840f-4e59-91b0-33114f4e42eb.png)

![image](https://user-images.githubusercontent.com/1594982/190428433-99b65c37-fb39-431b-8e24-1f59c9487f71.png)

I propose to separate the method name, its containing type, and its return type in many lines.

```
- Method(<args>)
- Return: <return_type>
- Member of: <containing_type>
```

It would be better if we have an option to seperate the list of arguments as well because the list can be very long, or the type name of an argument can be very long too (as in the second image).

**Comment by wmjordan** [comment]
> the type name of an argument can be very long too

I will only show the type name--without its namespace or containing type. The containing type or namespace can be displayed when we hover our mouse onto the type name.

So, your second example may look like

**Push&lt;T>**(....._parameters here_.....)
Returns: AsyncProcessHandle
Method of: ScreenContainer

**Comment by wmjordan** [comment]
Implemented in version 6.6
